### ISSUE TITLE
TTv5 Q&A: Label IDs are not unique for input elements.

### ISSUE THREAD

**Issue by HD508Github** [Section508Coordinators/DHSA-TT-200-A#155]
Location: Trusted Tester Q&A - June 18, 2019 page
https://dev.section508testing.net/mod/forum/post.php?forum=33
Issue: Under the Attachment section, there's a feature that allows users to drag/drop files by clicking within the Files area.  Afterwards, a 'File Picker' window displays allowing the user to select from various options.  Selecting the 'Wikimedia' option, there are two input fields: **Max image width (px)**: and Max image height **(px):**.  ANDI Output has identified these two input text fields as having duplicate IDs.  Also, the ANDI Accessibility Component/Label for each of these input text elements are the same (i.e., they both read: "Max image width (px)".

![Input Text FIelds_Duplicate Labels (File Picker window)](https://user-images.githubusercontent.com/34749031/59785331-81149100-9292-11e9-8cf0-07ea7a43a47c.jpg)

**Comment by drewnielson** [comment]
Confirmed that IDs for both fields ="undefined" and both labels are referencing the "undefined" id. @katbecker, this is another one that we'll need to send up the chain. Again, we should probably have some conversation with Cindy before doing so.

**Comment by katbecker** [comment]
Yes. This is a core admin function (versus something with the forum). Side note: if we are updating to core 3.7 we may want to wait to see if some of these things have been addresses.

**Comment by drewnielson** [comment]
Agreed. Let's hold off until after we upgrade and re-test these two issues. Confirmed with Moonami yesterday that we are upgrading to version 3.5 (not 3.7 yet).
